analyzers/src/SonarAnalyzer.CSharp/Rules/ExceptionsShouldBeLogged.cs

```diff
@@ -110,13 +110,30 @@ public override void VisitInvocationExpression(InvocationExpressionSyntax node)
             base.VisitInvocationExpression(node);
         }
 
+        private static IEnumerable<ISymbol> GetArgumentSymbolsDerivedFromException(InvocationExpressionSyntax invocation, SemanticModel semanticModel)
```

Comment:
The method is only referenced once, and the caller only takes the first result, so let's return an `ISymbol` instead of `IEnumerable<ISymbol>`.